dbe/domain-eng.yml

```diff
@@ -133,10 +133,10 @@ responses:
       - text: |
           Reply with one of the words in BOLD to continue
```

Comment:
Won't this change mean it's not bold anymore?

Reply:
Yup, I asked about it but was told to stick with what's in the sheet.